src/ui/public/filter_bar/filter_pill/filter_pill.html

```diff
@@ -51,6 +55,8 @@
       ng-focus="pill.activateActions()"
       ng-blur="pill.deactivateActions()"
       aria-label="Invert filter"
+      tooltip="Invert filter"
```

Comment:
Once the filter is inverted, maybe the action should change to "Un-invert filter"? We should probably change the `aria-label` to match this. I think this will clarify things for screen-readers, but also for sighted users since the signifiers for inversion aren't very clear to me (plus magnifying glass with red background vs minus magnifying glass with blue background).